### DISCUSSION TITLE
Revision History For Template Partials

### DISCUSSION THREAD

**Discussion by ciaran-framework-design** [ExpressionEngine/ExpressionEngine#4392] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
A clear and concise description of what you want to happen. Add any considered drawbacks.

**Describe alternatives you've considered**

**Teachability, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this and possibly write out a version the docs. Some of the best features are born from writing documentation first. 

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by ciaran-framework-design** [comment]
I store blocks of html in Template partials so having revision history would be helpful.

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4456
